doc/README.md

```diff
@@ -20,6 +20,7 @@ We also provide stand-alone packages that extend the Flame functionality:
   our own `Box2D` port called `Forge2D`.
 - [flame_tiled](https://pub.dev/packages/flame_tiled) Which provides integration with the
   [tiled](https://pub.dev/packages/tiled) package.
+- [flame_svg](https://pub.dev/packages/flame_svg) Add SVG rendering support for the Flame game engine.
```

Comment:
This sentence should be written in the same style as the other sentences in this list.
The package maps to `flutter_svg`.

Reply:
Fixed! Also committed your suggestions